rclpy/test/test_executor.py

```diff
@@ -29,7 +29,8 @@ class TestExecutor(unittest.TestCase):
     def setUp(self):
         self.context = rclpy.context.Context()
         rclpy.init(context=self.context)
-        self.node = rclpy.create_node('TestExecutor', namespace='/rclpy', context=self.context)
+        self.node = rclpy.create_node(
+            'TestExecutor', namespace='/rclpy', context=self.context)
```

Comment:
we would want to avoid unnecessary change if it is not required to pass the tests?